server/hub.go

```diff
@@ -20,6 +20,10 @@ import (
 	"github.com/tinode/chat/server/store/types"
 )
 
+// Request latency distribution bounds (in milliseconds).
+// "var" because Go does not support array constants.
+var RequestLatencyDistribution = [...]float64 {1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000,}
+
```

Comment:
Why use an array if you later convert it to slice anyway? E.g. `[...]float64{}` -> `[]float64{}`?

Reply:
Comma before `}` is not needed. `gofmt -l -w`